lib/bulk/unordered.js

```diff
@@ -160,54 +166,23 @@ function executeBatch(bulkOperation, batch, options, callback) {
     // If we have and error
     if (err) err.ok = 0;
     if (err instanceof MongoWriteConcernError) {
-      return handleMongoWriteConcernError(batch, bulkOperation.s.bulkResult, false, err, callback);
+      return handleMongoWriteConcernError(batch, bulkOperation.s.bulkResult, true, err, callback);
```

Comment:
the change to `true` here is for the `ordered` parameter. I looked into the code and this just forwards to `mergeBatchResults` which takes an _unused_ `ordered` parameter first. Should we take the time here to go remove all of that since it wasn't being used, and doesn't convey any useful information?